src/Infrastructure.Dapper/Repositories/UserRepository.cs

```diff
@@ -164,7 +183,50 @@ public async Task<IEnumerable<User>> GetManyAsync(IEnumerable<Guid> ids)
                 new { Ids = ids.ToGuidIdArrayTVP() },
                 commandType: CommandType.StoredProcedure);
 
+            UnprotectData(results);
             return results.ToList();
         }
     }
+
+    private void ProtectData(User user)
+    {
+        if (user == null)
+        {
+            return;
+        }
+
+        user.MasterPassword = user.MasterPassword == null ? null :
```

Comment:
❓ Do we use `?=` anywhere? Cleaner readout to me.

Reply:
I wasn't aware of `??=`. TIL. Let me see if it can help here.

Reply:
Still new to this operator, but not immediately clear to me how it could be utilized here. Any suggestion?

Reply:
I think I would just flip the logic to prefer your concatenated string first and then coalesce.